app/scripts/components/downloader/downloader.directive.ts

```diff
@@ -121,14 +135,15 @@ module ngApp.components.downloader.directive {
           }
         } else if (cookieStillThere()) {
           if (++attempts % timeoutInterval === 0) {
-            notify.closeAll();
-            notify({
-              message: null,
-              messageTemplate: (altMessage && attempts > timeoutInterval * 2) ? detailedMessage : simpleMessage,
-              container: '#notification',
-              classes: 'alert-warning',
-              scope: notifyScope
-            });
+            if (! alreadyShowingAltMessage && shouldShowAltMessage() && ! noToastMessageInDisplay()) {
```

Comment:
I think I've seen that somewhere but I can't remember anymore. I'll ask Terry next time I see him :)